data/init-cli/docker_init.yaml

```
@@ -197,6 +204,30 @@ examples: |-
   Your application will be available at http://localhost:8000
   ```
 
+  ### Example of selecting PHP with Apache
+  
+  The following example shows the prompts that appear after selecting `PHP with Apache` and example input. The PHP with Apache template is suitable for both pure PHP applications and applications using Composer as a dependency manager.
+  
+  ```console
+  ? What application platform does your project use? PHP with Apache
+  ? What version of PHP do you want to use? 8.2
+  ? What's the relative directory (with a leading .) for your app? ./src
+  ? What local port do you want to use to access your server? 9000
+  
+  CREATED: .dockerignore
+  CREATED: Dockerfile
+  CREATED: compose.yaml
+  CREATED: README.Docker.md
+  
+  ✔ Your Docker files are ready!
+  
+  Take a moment to review them and tailor them to your application.
+  
+  When you're ready, start your application by running: docker compose up --build
+  
+  Your application will be available at http://localhost:9000
```

all of the templates also have a new line related to the readme after this one:
```suggestion
  Your application will be available at http://localhost:9000
  
  Consult README.Docker.md for more information about using the generated files.
```